src/commands.ts

```diff
@@ -8,25 +8,30 @@ import * as pify from 'pify';
 import * as vscode from 'vscode';
 import Config from './config';
 import Utils from './utils';
+import { IContextMenuPaths } from './types';
+
 
 /* COMMANDS */
 
-async function open ( basePath ) {
+async function open ( passedBasePath: IContextMenuPaths | undefined ) {
```

Comment:
I don't like the name of this variable, **passed**BasePath. That's one of the reasons why sometimes I don't bother with types, one can't just assign a `string` to such variable and have the type system consider it just a `string` from that point onwards. 

Also wouldn't it be clearer to write `passedBasePath?: IContextMenuPaths` rather than explicitly using union types? 

And why is the interface called IContextMenuPath**s**, plural, if it's assigned to passedBasePath, singular?

Reply:
Variable name should change after keybinding will be able to accept `args`.